### DISCUSSION TITLE
Spende im JVerein Menüpunkt

### DISCUSSION THREAD

**Discussion by dippeal** [openjverein/jverein#125] [Polls]
Hallo,
im OpenJVerein ist als Menüpunkt der Spendenaufruf von Heiner drin. Brauchen wir den für OpenJVerein noch oder kann der Weg?
Eure Meinung in der Umfrage.

**Comment by willuhn** [comment]
Das Beste wäre ja die Gründung eines OpenJVerein Vereins ;)
Der könnte dann auch ein Spendenkonto haben.
